deno_dist/middleware/secure-headers/index.ts

```diff
@@ -1,29 +1,40 @@
+import { Buffer } from "node:buffer";
 import type { Context } from '../../context.ts'
 import type { MiddlewareHandler } from '../../types.ts'
 
+declare module '../../context.ts' {
+  interface ContextVariableMap {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    secureHeadersNonce?: string
+  }
+}
+
+type ContentSecurityPolicyOptionHandler = (ctx: Context, directive: string) => string
```

Comment:
Is the second parameter `directive: string` necessary? Or should it be exported for a user?

Reply:
I want users to be able to change their behaviour using directive, so I added export with the e184f6a change!

Reply:
👍